screens/auth/SignUpScreen.js

```diff
@@ -270,6 +334,49 @@ export default class SignUpScreen extends React.Component {
 
     return (
       <ScrollView showsVerticalScrollIndicator={false}>
+        <Modal
+          animationType={'slide'}
+          transparent={false}
+          visible={this.state.modalVisible}
+          onRequestClose={() => {
+            alert('Modal has been closed.');
```

Comment:
Definitely gonna need better error handling on these eventually. If I accidentally quit the captcha, I can't try to sign up again because the button stays disabled.

Reply:
Also depends on error state designs/messaging

Reply:
This is because the create account button clears the state of the form but leaves the text on screen, I'll work out where to move that functionality to fix this.